sirepo/pkcli/static_files.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""generate static files
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdlog, pkdp
+import pykern.pkio
+import sirepo.quest
+import sirepo.resource
+
+
+_ROOT_FILES = frozenset(("static/img/favicon.ico", "static/img/favicon.png"))
+
+
+def gen(target_d):
```

Comment:
target_d seems different from our usual naming conventions, why not target_dir?

Reply:
02365a7bf344032fd1501de3611900d9f39b590c